druntime/src/rt/cast_.d

```diff
@@ -23,10 +23,11 @@ pure:
 // but we are trying to implement dynamic cast.
 extern (D) private bool areClassInfosEqual(scope const ClassInfo a, scope const ClassInfo b) @safe
 {
-    if (a is b)
```

Comment:
I'd keep that guy; if for no other reason than it avoids an indirection and probably sufficient in the vast majority of situations.

Reply:
ok